website/_sass/color_schemes/_common.scss

```diff
@@ -1,5 +1,25 @@
-footer > img#logo {
+// Add support for external anchor icons.
```

Comment:
This "style" of writing sounds like an XXX to me. WDYT about changing it to: "Adds support for...". Not sure if that is the right way to go, but maybe it makes more sense 🤔.

Reply:
We could meet in the middle and use `/*`-style comments here? No strong opinion though. (I agree that the grammar is correct as-is!)

Reply:
There is a difference in behavior between `//`-style and `/*` style commments, though. `//` do not get compiled to CSS and `/*` do end up in the compiled CSS. See [SASS documentation](https://sass-lang.com/documentation/syntax/comments). Not sure whether it is worth it while sending more bytes over the line.